lib/hash.cc

```diff
@@ -293,15 +267,14 @@ static int hash_primes[] = {
     65357
 };
 
-int
-hashPrime(int n)
-{
+uint32_t
+hash_table::hashPrime(uint32_t n) {
```

Comment:
Please keep the Squid coding style with brackets on a separate line.
```suggestion
hash_table::hashPrime(uint32_t n)
{
```